SpinCore_pp/config_parser_fn.py

```diff
@@ -199,43 +205,6 @@ class configuration(object):
             None,
             "number of IR experiments collected in the ODNP experiment",
         ),
-        # reviewed to here
```

Comment:
I did this -- rather than adding new counters, I just did the counter thing below.  It seems to pass the config example.

Reply:
In testing AG realized the counter does increment properly but does not show up in the active.ini

Reply:
That would need debugging. I would first check if you are not writing the config at the end or something like that. Because the example for the config file does show the counter increments between runs, and it needs to save to the ini file to do that.

Reply:
Yes, this has to be a problem with the way that you're implementing it... Please see the example where I just added a counter that I call bogus counter that is entirely new and gets saved to the file

Reply:
I just pushed this to this branch

Reply:
I just pushed this to this branch

Reply:
I was wrong [here](https://github.com/jmfrancklab/FLinst/pull/14#discussion_r1734648342) and we do need to increment inside this file